packages/synapse-react-client/src/components/EntityDownloadConfirmation/EntityDownloadConfirmation.tsx

```diff
@@ -68,7 +68,7 @@ export function EntityDownloadConfirmation({
     onIsLoadingChange(isLoading)
   }, [isLoading, onIsLoadingChange])
 
-  if (isLoading || (!isLoading && entity)) {
+  if (isLoading) {
```

Comment:
What happens if the entity ID points to a FileEntity or a Docker repo (for example)?
Does the QueryWrapperTableDownloadConfirmation handle this case properly?

Reply:
<img width="1057" alt="Screenshot 2025-04-15 at 12 02 32 PM" src="https://github.com/user-attachments/assets/e3c7a2dc-57bf-41de-8914-4c17a37f05d0" />
you get an error message like this if the entity type isn't compatible